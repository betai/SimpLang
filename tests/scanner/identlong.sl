let main jfhvjfhfhgiusfdguhsdfguiieufgudifgniudfguibdsfgjsdfiugeiugneiounuefnguoenguoehguoehroughuoeghrouiehrguherguheguioherguherguheguioerhguerhguehrguheurgoiheurgiohueiorghuioerhgueohgruehrguoherguioherugiohreouigheorugherouhguoreihgueiohgehrgoiuehrguheroiughreuoghiouerhguorehguoherogheougheouhgeorghoerhgouerhgouerhgoerhgoehgeoherogheoghoehrgoergoherogeorghoergoegoergoerge =
  123
end
