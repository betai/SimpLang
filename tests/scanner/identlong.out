keyword let
identifier main
identifier jfhvjfhfhgiusfdguhsdfguiieufgudifgniudfguibdsfgjsdfiugeiugneiounuefnguoenguoehguoehroughuoeghrouiehrguherguheguioherguherguheguioerhguerhguehrguheurgoiheurgiohueiorghuioerhgueohgruehrguoherguioherugiohreouigheorugherouhguoreihgueiohgehrgoiuehrguheroiughreuoghiouerhguorehguoherogheougheouhgeorghoerhgouerhgouerhgoerhgoehgeoherogheoghoehrgoergoherogeorghoergoegoergoerge
operator =
integer 123
keyword end
